packages/next/shared/lib/utils.ts

```diff
@@ -404,7 +404,8 @@ export const SP = typeof performance !== 'undefined'
 export const ST =
   SP &&
   typeof performance.mark === 'function' &&
-  typeof performance.measure === 'function'
+  typeof performance.measure === 'function' &&
+  typeof performance.getEntriesByName === 'function'
```

Comment:
I never expect that there will be a browser supporting `performance.measure` but no `performance.getEntriesByName`.

Since the checklist is getting big, we might need to shrink it down a bit:

```ts
export const ST =
  SP && (['mark', 'measure', 'getEntriesByName'] as const).every(method => typeof performance[method] === 'function')
```

Reply:
Updated.